client/lib/form-state/index.js

```diff
@@ -304,7 +303,7 @@ function isInitialized( field ) {
 }
 
 function isEveryFieldInitialized( formState ) {
-	return every( formState, isInitialized );
```

Comment:
These seem risky to me because the usage is non-local but I checked and it looked like these functions are only called once each both in cases that seem safe. Is that matching what you found @tyxla?

Reply:
Yeah, those seem good, as they're always used with objects. I've also tested a couple of locations where these occur (the form to add a new DNS record and domain contact details form), and things work just like before.